mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb

```diff
@@ -2433,6 +2451,38 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 		}
 	}
 
+	if names, ok := d.GetOk("additional_pod_ranges_config.0.pod_range_names"); ok {
```

Comment:
When creating the cluster, the additional_pod_ranges_config can be included inside [IPAllocationPolicy](https://github.com/GoogleCloudPlatform/magic-modules/blob/0102b22bbd392fb6c19e7908f6c08ed7be9c9904/mmv1/third_party/terraform/services/container/resource_container_cluster.go.erb#L4358) for the CREATE request and the separate UPDATE request is not needed.

Reply:
Since this field is update-only, this separate update call upon creation needs to stay.